@here/harp-mapview/lib/MapView.ts

```diff
@@ -2432,18 +2432,9 @@ export class MapView extends THREE.EventDispatcher {
             this.m_forceCameraAspect !== undefined ? this.m_forceCameraAspect : width / height;
         this.setFovOnCamera(this.m_options.fovCalculation!, height);
 
-        // When calculating clip planes account for the highest building on the earth,
-        // multipling its height by projection scalling factor. This approach assumes
-        // constantHeight property of extruded polygon technique is set as default false,
-        // otherwise the near plane margins will be bigger then required, but still correct.
-        const projectionScale = this.projection.getScaleFactor(this.camera.position);
-        const maxHeight = EarthConstants.MAX_BUILDING_HEIGHT * projectionScale;
-        // Copy all properties from new view ranges to our readonly object.
-        // This allows to keep all view ranges references valid and keeps up-to-date
-        // information within them. Works the same as copping all properties one-by-one.
         Object.assign(
             this.m_viewRanges,
-            viewRanges === undefined ? this.m_visibleTiles.updateClipPlanes(maxHeight) : viewRanges
+            viewRanges === undefined ? this.m_visibleTiles.updateClipPlanes() : viewRanges
```

Comment:
Consider what may happen when decoder will not calculate maxBuildingHeight - then we leave clip planes without accounting for any buildings. The base approach though would be to have tile.maxGeometryHeight set to EarthConstants.MAX_BUILDING_HEIGHT by default. See also my comments below.